depends/packages/native_libmultiprocess.mk

```diff
@@ -1,8 +1,8 @@
 package=native_libmultiprocess
-$(package)_version=8da797c5f1644df1bffd84d10c1ae9836dc70d60
+$(package)_version=003eb04d6d0029fd24a330ab63d5a9ba08cf240f
```

Comment:
Looks like this can be bumped further now that https://github.com/chaincodelabs/libmultiprocess/pull/98 has been merged?

Reply:
Bumped even further in https://github.com/bitcoin/bitcoin/pull/30513 :)